leases/tests/test_lease_models.py

```diff
@@ -175,3 +175,57 @@ def test_winter_storage_leases_should_not_overlap(winter_storage_place):
         )
 
     assert "WinterStoragePlace already has a lease" in str(exception.value)
+
+
+def test_berth_lease_inactive_berth_raises_error(berth):
+    berth.is_active = False
+    berth.save()
+
+    with pytest.raises(ValidationError) as exception:
+        BerthLeaseFactory(berth=berth)
+
+    assert "Selected berth is not active" in str(exception.value)
+
+
+def test_berth_lease_inactive_berth_updating(berth_lease, berth):
+    berth.is_active = False
+    berth.save()
+
+    assert BerthLease.objects.get(id=berth_lease.id).berth != berth
+
+    berth_lease.berth = berth
```

Comment:
hm, I think we should also disallow changing berth on a berth lease... our flow supposes that if we want to change the berth, we generate a new lease and delete the one generated before.

Reply:
Oe, really good catch! We should definitely add some validation to avoid places being changed.

Reply:
Moved to [VEN-569](https://helsinkisolutionoffice.atlassian.net/browse/VEN-569)